tests/unit/modin/test_unsupported.py

```diff
@@ -135,20 +135,6 @@ def test_unsupported_df(df_method, kwargs):
         getattr(mock_df, df_method)(**kwargs)
 
 
-@pytest.mark.parametrize(
-    "df_method, kwargs",
-    [["items", {}], ["iteritems", {}]],
-)
-def test_unsupported_df_generator(df_method, kwargs):
```

Comment:
- pandas 2.0 [removed](https://pandas.pydata.org/docs/whatsnew/v2.0.0.html#removal-of-prior-version-deprecations-changes) `iteritems`, an alias for `items`, in pandas 2.0, so no need to include it here.
- We now support `items`.